bugbear.py

```diff
@@ -1759,7 +1783,12 @@ def visit_Lambda(self, node):
         " flag."
     )
 )
-
+B908 = Error(
+    message=(
```

Comment:
We should also describe the new warning in https://github.com/pycqa/flake8-bugbear#list-of-warnings. I think to many people it may not be obvious why this is a problem, so we should have a clear explanation.